src/i18n/strings/es.ts

```diff
@@ -159,7 +158,9 @@ export const templates = {
   sb9e8a9bfc8bcf398: `Rhode Island`,
   sba0e115b257c6647: `una bomba de calor de fuente de aire`,
   sbafed52303428041: `https://homes.rewiringamerica.org/es/federal-incentives/home-efficiency-rebates`,
+  sbb254e3763405cb1: `Al compartir tu correo electrónico, aceptas nuestros`,
```

Comment:
`aceptas` –> `acepta`

Reply:
Thanks, this is my original, now outdated change -- I've updated to `acepta`!